po/it.po

```diff
@@ -1,5 +1,671 @@
+#
+# XP312 < >, 2020.
+# Saverio <saverio.morelli@outlook.it>, 2021.
+#
 msgid ""
 msgstr ""
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
-"Plural-Forms: nplurals=2; plural=n != 1;\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+"Project-Id-Version: unnamed project\n"
+"Last-Translator: Saverio <saverio.morelli@outlook.it>\n"
+"Language-Team: Italian <saverio.morelli@outlook.it>\n"
+"Language: it\n"
+"Content-Transfer-Encoding: 8bit\n"
+"PO-Revision-Date: 2021-06-28 03:30+0200\n"
+"X-Generator: Gtranslator 40.0\n"
+
+#: Dialogs/ExportDialog.vala:151
+msgid "Export to:"
+msgstr "Esporta come:"
+
+#: Dialogs/ExportDialog.vala:157
+msgid "Select Folder"
+msgstr "Seleziona la cartella"
+
+#: Dialogs/ExportDialog.vala:166
+msgid "Format:"
+msgstr "Formato:"
+
+#: Dialogs/ExportDialog.vala:180
+msgid "Quality:"
+msgstr "Qualità:"
+
+#: Dialogs/ExportDialog.vala:192
+msgid "Compression:"
+msgstr "Compressione:"
+
+#: Dialogs/ExportDialog.vala:206
+msgid "Transparency:"
+msgstr "Trasparenza:"
+
+#: Dialogs/ExportDialog.vala:219
+msgid "Scale:"
+msgstr "Scala:"
+
+#: Dialogs/ExportDialog.vala:243 FileFormat/FileManager.vala:49
+#: FileFormat/FileManager.vala:105
+msgid "Cancel"
+msgstr "Annulla"
+
+#: Dialogs/ExportDialog.vala:250 Dialogs/ShortcutsDialog.vala:63
+#: Layouts/HeaderBar.vala:112
+msgid "Export"
+msgstr "Esporta"
+
+#: Dialogs/ReleaseDialog.vala:47 Dialogs/SettingsDialog.vala:217
+msgid ""
+"WARNING!\n"
+"Akira is still under development and not ready for production. Missing "
+"features, random bugs, and black holes opening in your kitchen are to be "
+"expected."
+msgstr ""
+"ATTENZIONE!\n"
+"Akira è ancora in fase di sviluppo e non è pronta per la produzione. "
+"Potrebbero verificarsi bug casuali, funzionalità mancanti, buchi neri "
+"durante l'apertura nella tua cucina."
+
+#: Dialogs/ReleaseDialog.vala:100 Dialogs/SettingsDialog.vala:249
+msgid "Make a Donation"
+msgstr "Fai una donazione"
+
+#: Dialogs/ReleaseDialog.vala:109 Dialogs/SettingsDialog.vala:258
+msgid "Suggest Translations"
+msgstr "Suggerisci traduzione"
+
+#: Dialogs/ReleaseDialog.vala:118 Dialogs/SettingsDialog.vala:267
+msgid "Report a Problem"
+msgstr "Segnala un problema"
+
+#: Dialogs/SettingsDialog.vala:40
+msgid "Preferences"
+msgstr "Preferenze"
+
+#: Dialogs/SettingsDialog.vala:50 Dialogs/SettingsDialog.vala:73
+#: Dialogs/ShortcutsDialog.vala:45
+msgid "General"
+msgstr "Generale"
+
+#: Dialogs/SettingsDialog.vala:51 Dialogs/SettingsDialog.vala:86
+msgid "Interface"
+msgstr "Interfaccia"
+
+#: Dialogs/SettingsDialog.vala:52 Layouts/HeaderBar.vala:280
+msgid "Shapes"
+msgstr "Figure"
```

Comment:
Non so se "Figure" sia corretto.
La traduzione letterale è "Forme" ma non so se si capisca.
Puoi controllare Figma come traduce "Shapes tool"?

Reply:
@Alecaddd "Figura" è nell'almbito della geometria, mentre "forma" nell'ambito dell'aspetto.
Anche a me ha messo un po' di dubbi su come tradurre questo termine... poi, però, ho pensato che, in effetti, apre un sotto-menu con "rettangolo" ed "ellisse", che sono di per sé delle figure geometriche.
Non so se il ragionamento quadra

Reply:
Si, ho avuto lo stesso dubbio. Direi che va più che bene cosí.
Il ragionmaneto "cerchia" :P